.github/workflows/sasjs-tests.yml

```diff
@@ -0,0 +1,87 @@
+# This workflow will do a deploy of testing framework with current version of adapter installed.
+# It will run the cypress that will run the tests and see if all of them are passing.
+
+name: SASjs Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      
+      - name: Install SSH Key
+        uses: shimataro/ssh-key-action@v2
+        with: 
+          key: ${{ secrets.DCGITLAB_KEY }}
+          known_hosts: 'placeholder'
+
+      - name: Write VPN Files
+        run: |
+          echo "$CA_CRT" > .github/vpn/ca.crt
+          echo "$USER_CRT" > .github/vpn/user.crt
+          echo "$USER_KEY" > .github/vpn/user.key
+          echo "$TLS_KEY" > .github/vpn/tls.key
+        shell: bash
+        env:
+          CA_CRT: ${{ secrets.CA_CRT}}
+          USER_CRT: ${{ secrets.USER_CRT }}
+          USER_KEY: ${{ secrets.USER_KEY }}
+          TLS_KEY: ${{ secrets.TLS_KEY }}
+
+      - name: Install Open VPN
+        run: |
+          sudo apt install apt-transport-https
+          sudo wget https://swupdate.openvpn.net/repos/openvpn-repo-pkg-key.pub
+          sudo apt-key add openvpn-repo-pkg-key.pub
+          sudo wget -O /etc/apt/sources.list.d/openvpn3.list https://swupdate.openvpn.net/community/openvpn3/repos/openvpn3-bionic.list
+          sudo apt update
+          sudo apt install openvpn3
+
+      - name: Start Open VPN 3
+        run: openvpn3 session-start --config .github/vpn/config.ovpn
+
+      - name: Deploy sasjs-tests
+        run: | 
+          npm i
+          npm install -g replace-in-files-cli
+          cd sasjs-tests
+          replace-in-files --regex='"@sasjs/adapter".*' --replacement='"@sasjs/adapter":"2.8.13",' ./package.json
```

Comment:
Should this not package and install the adapter from the current branch?

Reply:
yes - it shouldn't be hardcoded

Reply:
It is doing it later yes. But then instead of the version in package.json path to adapter build package is saved. At that moment of the script that package does not exist so `npm install` fails. I am putting a hardcoded version that for sure exists on npm, so npm can proceed without fail. Later in the script, an adapter version from that branch is built and installed for the sasjs-tests.